includes/abstracts/abstract.llms.shortcode.course.element.php

```diff
@@ -77,18 +89,22 @@ protected function get_output() {
 
 		}
 
+		if ( ! $current_id && llms_is_editor_block_rendering() ) {
+			$current_id = LLMS_Shortcodes_Blocks::get_placeholder_course_id();
+		}
+
```

Comment:
instead of mixing concepts, can't we have filters so that FROM the blocks shortcode we alter the placeholder course id from there?...